### DISCUSSION TITLE
Custom Loading Screen

### DISCUSSION THREAD

**Discussion by em9836** [melonjs/melonJS#1244] [Q&A] [answered]
So I am new to melonjs, and I want to make my own Loading Screen instead of just using the default one. I have made attempts, but it didn't really work well. any ideas on how I should make it?

**Comment by obiot** [comment, marked as answer]
it's supposed to be actually quite simple (when it works).

Define a new Loading Screen that extend the `Stage` class (see the source code of our loading screen [here](https://github.com/melonjs/melonJS/blob/master/packages/melonjs/src/loader/loadingscreen.js)), and then just "assign" it to the loading screen by calling `state.set(state.LOADING, new youCustomLoadingClass());`

where are you stuck exactly ?

**Reply by em9836** [reply]
I already made my Loading Screen, just trying to get the game to use it was where I was stuck. I will try this though. thank you!

**Reply by em9836** [reply]
It worked! Thank you!
